Source/Core/Core/HW/DSPHLE/DSPHLE.cpp

```diff
@@ -191,26 +191,37 @@ void DSPHLE::DSP_WriteMailBoxLow(bool cpu_mailbox, u16 value)
 // Other DSP functions
 u16 DSPHLE::DSP_WriteControlRegister(u16 value)
 {
-  DSP::UDSPControl temp(value);
```

Comment:
When in doubt, consider it the latter :P
The DSP has a lot of quirky features that _should_ make things like that possible (or rather, even guarantee it) - but its been too long for me to remember all the details.
In LLE, its just a `u16` (and not a reference or anything), so the write at the end (atomically?) puts the modified value in. If you write it first and use `g_dsp.cr` all the time it would end up leaking the intermediate steps to the outside.
But dunno, maybe someone else (skid, delroth?) has a better idea on what to do here. Maybe it doesn't matter at all, but you never know.